lib/helpers.ts

```diff
@@ -168,4 +169,28 @@ export function sortStageOptions(options:ExploreOptionType[]){
 
 export function clamp(x:number, lower:number, upper:number) {
     return Math.max(lower, Math.min(x, upper));
+}
+
+export function parseRawAssayType(t:string) {
+    // It comes in the form bts:CamelCase-NameLevelX (may or may not have that hyphen).
+    // We want to take that and spit out { name: "Camel Case-Name", level: "Level X" }
+    //  (with the exceptions that "RNA" is one word and "ScRNA" parses to "scRNA")
+
+    // See if there's a Level in it
+    const splitByLevel = t.split("Level");
+    const level = splitByLevel.length > 1 ? `Level ${splitByLevel[1]}` : null;
+    const extractedName = splitByLevel[0].match(/bts:(.+)/);
+    if (extractedName && extractedName[1]) {
+        // Convert camel case to space case
+        // Source: https://stackoverflow.com/a/15370765
+        let name = extractedName[1].replace( /([A-Z])([A-Z])([a-z])|([a-z])([A-Z])/g, '$1$4 $2$3$5' );
+
+        // Handle special case: scRNA
+        name = name.replace(/Sc RNA\b/g, "scRNA");
```

Comment:
prolly anything prefixed with `Sc ` should be changed in this way 

e.g. `Sc ATAC-seq` should be `scATAC-seq`

Reply:
sc is short for single cell

Reply:
same applies to `sn` (single nucleus), see data standards list here:

https://htan-portal-nextjs-ln46lfjfo.vercel.app/standards

Reply:
okay cool, Ill just always treat a lone sc as a prefix